src/shared/components/palette/document/write-document-field.component.ts

```diff
@@ -144,6 +144,12 @@ export class WriteDocumentFieldComponent extends AbstractFieldWriteComponent imp
 
         this.valid = true;
         this.fileUploadStateService.setUploadInProgress(false);
+        // refresh replaced document info
+        if (this.caseField.value) {
+          this.caseField.value.document_binary_url = document._links.binary.href;
```

Comment:
Do you need to check for `document._links.binary` also?
```
if (this.caseField.value && document._links.binary) {
//
```

Reply:
thanks @udaydenduluri33 , nope as the document._links.binary is coming from the api body, and it will never has null/undefined in that nested objects. It is same as line 140 to 142.

Reply:
OK